src/classes/DatabaseManager.js

```diff
@@ -168,7 +169,7 @@ class DatabaseManager {
    */
```

Comment:
Your return would imply it returns an array of promises. Maybe change it to `Promise(Object)` which means a promise which resolves with an Object, which is what createTable returns.